src/components/TextAreaInput/TextAreaInput.js

```diff
@@ -88,12 +88,6 @@ function PrivateTextAreaInput({
     doValidation(ev.target.value, true)
   }
 
-  const onPaste = (ev) => {
-    const val = ev.clipboardData.getData('text/plain')
-    const restrictedVal = restrictIllegal ? restrict(val) : val
-    setValue(restrictedVal)
-  }
-
```

Comment:
Every onPaste event was was followed by an onChange event, causing setValue with restrictedVal twice.

Reply:
Pasting without the onPaste handler still calls onChange so the sanitization still occurs.

Reply:
:nod: